static/css/rue89.css

```diff
@@ -0,0 +1,134 @@
+a:link,
```

Comment:
The CSS file is indeed a compiled version of my SCSS files.
Most rules are CSS syntax anyway, as I am overriding the declarations from the compiled bundle.css file.
I could un-compile them and turn
```css
    .view-login header.global-new > nav .nav-courseware a.cta-login,
    .view-login header.global-new > nav .nav-courseware a.cta-register,
    .view-login header.global.slim > nav .nav-courseware a.cta-login,
    .view-login header.global.slim > nav .nav-courseware a.cta-register,
    .view-login header.global > nav .nav-courseware a.cta-login,
    .view-login header.global > nav .nav-courseware a.cta-register,
    .view-passwordreset header.global-new > nav .nav-courseware a.cta-login,
    .view-passwordreset header.global-new > nav .nav-courseware a.cta-register,
    .view-passwordreset header.global.slim > nav .nav-courseware a.cta-login,
    .view-passwordreset header.global.slim > nav .nav-courseware a.cta-register,
    .view-passwordreset header.global > nav .nav-courseware a.cta-login,
    .view-passwordreset header.global > nav .nav-courseware a.cta-register,
    .view-register header.global-new > nav .nav-courseware a.cta-login,
    .view-register header.global-new > nav .nav-courseware a.cta-register,
    .view-register header.global.slim > nav .nav-courseware a.cta-login,
    .view-register header.global.slim > nav .nav-courseware a.cta-register,
    .view-register header.global > nav .nav-courseware a.cta-login,
    .view-register header.global > nav .nav-courseware a.cta-register,
    body header.global-new > nav .nav-courseware a.cta-login,
    body header.global-new > nav .nav-courseware a.cta-register,
    body header.global.slim > nav .nav-courseware a.cta-login,
    body header.global.slim > nav .nav-courseware a.cta-register,
    body header.global > nav .nav-courseware a.cta-login,
    body header.global > nav .nav-courseware a.cta-register {
        background-color: #fff;
    }
```

back into something like 
```scss
	.view-login,
	.view-password,
	.view-register {
		header {
			.global,
			.global-new,
			.global.slim {
				> nav {
					.nav-courseware {
						a {
							.cta-login,
							.cta-register {
								background-color: #fff;
							}
						}
					}
				}
			}
		}
	}
```
but I'm not sure if there's a point in it.

I also considered simply using `!important`. This would clearly make the previous code less verbose : 
```css
.cta-login,
.cta-register {
    background-color: #fff !important;
}
```
It might be the best solution.

I'm open to discussion on this subject.